pkg/catalog/keycloak/keycloak.go

```diff
@@ -67,9 +71,12 @@ func (m *Manager) Info(instance engine.Container) (map[string]string, error) {
 	user := instance.Env["KEYCLOAK_ADMIN"]
 	password := instance.Env["KEYCLOAK_ADMIN_PASSWORD"]
 
+	uri := fmt.Sprintf("localhost:%d/admin", DefaultPort)
```

Comment:
hm, just spotted, multiple instances (why would one have that?) are not ideally supported with static ports. any basic principles how to deal with that in general?

Reply:
you might find this example useful:
https://github.com/adrianliechti/devkit/blob/main/pkg/catalog/postgres/postgres.go#L72

=> finding the current mapping (local port / static container port) from the runtime-context

Reply:
the console command is a nice hack too:

to expose-non exposed ports (not specified with -p), it starts a helper container with socat and a port-forward, which forwards traffic to the container (withing the vm on your mac/win)